functions/src/models/offers/offersWithLocation.ts

```diff
@@ -0,0 +1,83 @@
+import { FirestoreDataConverter } from '@google-cloud/firestore';
+import { IsArray } from 'class-validator';
+import { firestore } from 'firebase-admin';
```

Comment:
Given this is a CF only model this is OK, but we want our model classes to use regular firebase so they can later be shared by Web and CF based projects.

Reply:
right, but I am not fully sure if we can do that right now, I have the change for client side libraries in the models I used in the `web-client` project.